fitbenchmarking/utils/fitbm_result.py

```diff
@@ -1,13 +1,17 @@
 """
 FitBenchmarking results object
 """
+
+from statistics import fmean, harmonic_mean, median
 from typing import TYPE_CHECKING, Literal
 
-from statistics import median, harmonic_mean, fmean
 import numpy as np
 from scipy import stats
 
+from fitbenchmarking.controllers.base_controller import Controller
+from fitbenchmarking.cost_func.base_cost_func import CostFunc
 from fitbenchmarking.cost_func.nlls_base_cost_func import BaseNLLSCostFunc
+from fitbenchmarking.parsing.fitting_problem import FittingProblem
```

Comment:
If these are moving here, we can remove the `TYPE_CHECKING` guarded imports below.